examples/c5g7_transient/density_change/surfaces.py

```diff
@@ -0,0 +1,36 @@
+import openmc
+
+###############################################################################
+#                       Create the OpenMC Surfaces
+###############################################################################
+
+surfaces = {}
+
+# Instantiate ZCylinder surfaces
+surfaces['Fuel OR']            = openmc.ZCylinder(x0=0, y0=0, r=0.4095, name='Fuel OR')
+surfaces['Fuel Inner Clad IR'] = openmc.ZCylinder(x0=0, y0=0, r=0.4180, name='Fuel Inner Clad IR')
+surfaces['Fuel Inner Clad OR'] = openmc.ZCylinder(x0=0, y0=0, r=0.4750, name='Fuel Inner Clad OR')
+surfaces['Fuel Outer Clad IR'] = openmc.ZCylinder(x0=0, y0=0, r=0.4800, name='Fuel Outer Clad IR')
+surfaces['Fuel Outer Clad OR'] = openmc.ZCylinder(x0=0, y0=0, r=0.5400, name='Fuel Outer Clad OR')
+surfaces['Guide Tube IR']      = openmc.ZCylinder(x0=0, y0=0, r=0.3400, name='Guide Tube IR')
+surfaces['Guide Tube OR']      = openmc.ZCylinder(x0=0, y0=0, r=0.5400, name='Guide Tube IR')
```

Comment:
For z-cylinders at the origin, you don't need to specify x0 and y0
```suggestion
surfaces['Fuel OR']            = openmc.ZCylinder(r=0.4095, name='Fuel OR')
surfaces['Fuel Inner Clad IR'] = openmc.ZCylinder(r=0.4180, name='Fuel Inner Clad IR')
surfaces['Fuel Inner Clad OR'] = openmc.ZCylinder(r=0.4750, name='Fuel Inner Clad OR')
surfaces['Fuel Outer Clad IR'] = openmc.ZCylinder(r=0.4800, name='Fuel Outer Clad IR')
surfaces['Fuel Outer Clad OR'] = openmc.ZCylinder(r=0.5400, name='Fuel Outer Clad OR')
surfaces['Guide Tube IR']      = openmc.ZCylinder(r=0.3400, name='Guide Tube IR')
surfaces['Guide Tube OR']      = openmc.ZCylinder(r=0.5400, name='Guide Tube IR')
```